media/css/firefox/australis/growth-firstrun-test1.less

```diff
@@ -0,0 +1,112 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../../sandstone/lib.less";
+
+// customize FxA UI
+
+#fxa-background {
+    position: relative !important; // override inline style
+    padding: @baseLine 0;
+    display: none;
+
+    &.visible {
+        display: block;
+    }
+}
+
+#fxa {
+    top: auto !important; // it's inline on the element :/
+}
+
+// container for things that get moved into FxA overlay
+#fxa-extra {
+    display: none;
+}
+
+#fxa-value-prop {
+    overflow: hidden;
+    opacity: 0;
+    margin-bottom: @baseLine;
+    text-align: center;
+    transition: all 0.5s ease;
+
+    h2 {
+        font-size: 22px;
+    }
+
+    &.visible {
+        opacity: 1;
+    }
+}
+
+#fxa-value-prop-intro {
+}
+
+#fxa-value-prop-verified {
+    display: none;
+}
+
+.fxa-extra-content {
+    width: 900px;
+    margin: 0 auto;
+}
+
+// these CTAs come in after 30 seconds on the page
+// will change behavior when we can listen for clicks
+// on 'Signup' button
+#fxa-delayed-ctas {
+    display: block;
+    .clearfix();
+    opacity: 0;
+    list-style-type: none;
+    margin-top: -50px; // fxa iframe is a bit taller than it needs to be...
+    transition: opacity 0.7s ease;
+
+    &.visible {
+        opacity: 1;
+    }
+
+    &.noiframe {
+        margin-top: @baseLine * 3;
+    }
+
+    li {
+        .border-box();
+        float: left;
+        width: 33%;
+        padding: 0 @baseLine;
+        margin: 0;
+        text-align: center;
+        border-right: 1px solid #ddd;
+
+        &:last-child {
+            border-right: none;
+        }
+    }
+
+    strong {
+        .font-size(@largeFontSize);
+        display: block;
+        margin-bottom: @baseLine / 2;
+    }
+
+    a {
+        background: linear-gradient(#84C63C, #489615) repeat scroll 0% 0% #489615;
+        border-radius: 6px;
+        box-shadow: 0px 3px rgba(0, 0, 0, 0.1), 0px -4px rgba(0, 0, 0, 0.1) inset;
+        color: #FFF;
+        display: inline-block;
+        font: bold 16px "Trebuchet MS",sans-serif;
```

Comment:
Heh, it's the font used on the AMO iframed buttons, so just keeping visually consistent.